core/src/main/java/org/apache/calcite/rel/RelHomogeneousShuttle.java

```diff
@@ -19,6 +19,7 @@
 import org.apache.calcite.rel.core.TableFunctionScan;
 import org.apache.calcite.rel.core.TableScan;
 import org.apache.calcite.rel.logical.LogicalAggregate;
+import org.apache.calcite.rel.logical.LogicalCalc;
 import org.apache.calcite.rel.logical.LogicalCorrelate;
```

Comment:
Tips: remove the `.` in the commit message.

Reply:
Done. `.` is not in commit message anymore.